tools/devops/automation/templates/tests-stage.yml

```
@@ -327,6 +326,12 @@ stages:
     xqaCertPass: $(xqa--certificates--password)
     condition: ${{ parameters.runTests }}
     postPipeline: true
+    ${{ if eq(parameters.isPR, true) }}:
+      # Using Azure hosted pool for PR simulator tests
+      useAzHostedPool: true
+      testPool: 'macOS-latest'
```

Xcode fails to install:

> This Xcode version requires macOS ≥ 14.5 (currently running on 12.7.6)

```suggestion
      testPool: 'macOS-14'
```

macOS 12 is the latest according to https://github.blog/changelog/2024-01-30-github-actions-macos-14-sonoma-is-now-available/...